udao_trace/configuration/conf.py

```diff
@@ -131,6 +131,7 @@ def denormalize(
             raise Exception(
                 f"conf_norm should be within [0, 1], got {conf_norm} instead"
             )
+        eps = 1e-6
```

Comment:
Yes. The original eps does not work.

Reply:
The function `denormalize` together with the default eps has been tested as working. Can you bring an example when the original eps is not working? 

https://github.com/Angryrou/udao-spark-optimizer/blob/36f91054eed31347c89eecd0693e7b15d02e4ca6/udao_trace/tests/configuration/test_spark_conf.py#L141

Reply:
I commented on the line of `eps = 1e-6` and tested the current code which works fine. 

In the previous experiments, where I cannot recall which model we were using, it brought the denormalization issue and was fixed when I set it as `eps = 1e-6`. That's why I mentioned that " the original eps does not work". And from then on, the code kept this setting. 

Since now the function `denormalization` works fine, I will delete this line.